components/tab-tile/tab-tile.hbs

```diff
@@ -1,220 +1,338 @@
-        </div>
-    </div>
+            <div class="msds-tabs__tab msds-tabs__tab--disabled" data-index="1">
+                <div class="msds-tabs__tab-svg msds-reliable" title="Reliable"></div>
+                <div class="msds-tabs__tab-text">
+                    {{tab.placeholderHeading}}
+                </div>
+            </div>
 
-    <div class="msds-tab-tile msds-tab-tile--disabled">
-   <div class="msds-tab-tile__svg msds-download-illustration" title="Donwload Illustration"></div>
-        <div class="msds-tab-tile__text">
-            {{tab.placeholderHeading}}
-        </div>
-    </div>
+            <div class="msds-tabs__tab msds-tabs__tab--disabled" data-index="2">
+                <div class="msds-tabs__tab-svg msds-scalable" title="Scalable"></div>
+                <div class="msds-tabs__tab-text">
+                    {{tab.placeholderHeading}}
+                </div>
+            </div>
 
-    <div class="msds-tab-tile msds-tab-tile--disabled">
-   <div class="msds-tab-tile__svg msds-download-illustration" title="Donwload Illustration"></div>
-        <div class="msds-tab-tile__text">
-            {{tab.placeholderHeading}}
-        </div>
-    </div>
+            <div class="msds-tabs__tab msds-tabs__tab--disabled" data-index="3">
+                <div class="msds-tabs__tab-svg msds-end-user" title="End User"></div>
+                <div class="msds-tabs__tab-text">
+                    {{tab.placeholderHeading}}
+                </div>
+            </div>
 
-    <div class="msds-tab-tile msds-tab-tile--disabled">
-   <div class="msds-tab-tile__svg msds-download-illustration" title="Donwload Illustration"></div>
-        <div class="msds-tab-tile__text">
-            {{tab.placeholderHeading}}
+            <div class="msds-tabs__tab msds-tabs__tab--disabled" data-index="4">
+                <div class="msds-tabs__tab-svg msds-interconnect" title="Interconnect"></div>
+                <div class="msds-tabs__tab-text">
+                    {{tab.placeholderHeading}}
+                </div>
+            </div>
+
+            <div class="msds-tabs__tab msds-tabs__tab--disabled" data-index="5">
+                <div class="msds-tabs__tab-svg msds-xprotect" title="XProtect"></div>
+                <div class="msds-tabs__tab-text">
+                    {{tab.placeholderHeading}}
+                </div>
+            </div>
         </div>
+        <div class="msds-tabs__spacer"></div>
     </div>
-
 </div>
 
-<br>
-<div class="row-scroll-x-mobile">
-
-    <div tabindex="1" class="msds-tab-tile msds-tab-tile--active">
-   <div class="msds-tab-tile__svg msds-download-illustration" title="Donwload Illustration"></div>
-        <div class="msds-tab-tile__text">
-            {{tab.placeholderHeading}}
-        </div>
-    </div>
+<h2>2 Tabs</h2>
+<div class="msds-tabs">
+    <div class="msds-tabs__container">
+        <div class="msds-tabs__spacer"></div>
+        <div class="msds-tabs__tabs-container">
```

Comment:
and here we use msds-tabs__tabs-container

Reply:
wrapper for the tabs, the container is used in the js and to provide the spacers between the tab container and browser

Reply:
msds-tabs__container is main component div container for tab elements and spacers. msds-tabs__tabs-container is div container for tab elements. Two completly diffrent element for diffrent pruposes, what is a problem Danny?

Reply:
I tried to remove msds-tabs__container, but that breaks the sticky tabs so we are going to have to live with this implementation